mw-webapp/src/component/input/Input.module.scss

```diff
@@ -0,0 +1,30 @@
+@import "src/styles/variables.scss";
+
+.input_wrapper {
+  position: relative;
+  font-size: 1.4rem;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.visually-hidden {
+  position: fixed;
+  transform: scale(0);
+}
+
+.input {
+  letter-spacing: 0.03em;
+  outline: 0;
+  border: 1px solid $input-border-primary;
+  padding: 7px;
+  width: 420px;
```

Comment:
let's avoid to use hardcoded width for such elements like input, it could be used in different components, not only full-page forms